src/any_llm/providers/gateway/gateway.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any
+
+from any_llm.providers.openai.base import BaseOpenAIProvider
+
+
+class GatewayProvider(BaseOpenAIProvider):
+    ENV_API_KEY_NAME = "None"
```

Comment:
Shouldn't we:

```suggestion
    ENV_API_KEY_NAME = "GATEWAY_MASTER_KEY"
```

Reply:
Or maybe we very genetically name at Gateway API key